packages/core/entity/FileDetail/index.ts

```diff
@@ -156,62 +156,20 @@ export default class FileDetail {
         return path as string;
     }
 
-    public get cloudPath(): string {
-        //// REMOVE THIS (BACKWARDS COMPAT)
-        if (this.path.startsWith("/allen")) {
-            return this.convertAicsDrivePathToAicsS3Path(this.path);
-        }
-        ////
-
-        // AICS FMS files' paths are cloud paths
-        return this.path;
-    }
-
     public get localPath(): string | null {
-        //// REMOVE THIS (BACKWARDS COMPAT)
-        if (this.path.startsWith("/allen")) {
-            return this.path;
-        }
-        ////
-
-        if (this.getAnnotation("Cache Eviction Date")) {
-            return this.getLocalPath();
-        }
-        return null;
-    }
-
-    public getLocalPath(): string | null {
-        const localPrefix = NAS_HOST_PREFIXES[this.env];
         const cloudPrefix = `${AICS_FMS_S3_URL_PREFIX}${AICS_FMS_S3_BUCKETS[this.env]}`;
-
-        if (this.path.startsWith(cloudPrefix)) {
+        if (this.getAnnotation("Cache Eviction Date") && this.path.startsWith(cloudPrefix)) {
```

Comment:
I confirmed that all files in FSS with `should_be_in_local` also have a cache eviction date, so this should be safe to deploy 👍